pkg/ccl/backupccl/restore_online.go

```diff
@@ -173,8 +228,8 @@ func sendAddRemoteSSTWorker(
 				// span rather than one we have added to, since we add with estimated
 				// stats and splitting a span with estimated stats is slow.
 				if batchSize+file.BackupFileEntryCounts.DataSize > targetBatchSize {
-					log.Infof(ctx, "flushing %s batch of %d SSTs due to size limit up to %s in in span %s", sz(batchSize), len(toAdd), file.BackupFileEntrySpan.Key, entry.Span)
-					if err := flush(file.BackupFileEntrySpan.Key); err != nil {
+					log.Infof(ctx, "flushing %s batch of %d SSTs due to size limit up to %s in in span (old keyspace) %s", sz(batchSize), len(toAdd), file.BackupFileEntrySpan.Key, entry.Span)
```

Comment:
are we missing a new verb here?

also `in in`

Reply:
done.